gulpfile.js

```diff
@@ -1,8 +1,9 @@
-var gulp    = require('gulp');
-var order   = require('gulp-order');
-var uglify  = require('gulp-uglify');
-var concat  = require('gulp-concat');
-var rename  = require('gulp-rename');
+var gulp = require('gulp');
+var concat = require('gulp-concat');
+var rename = require('gulp-rename');
+var order = require('gulp-order');
+var uglify = require('gulp-uglify');
+var sourcemaps = require('gulp-sourcemaps');
```

Comment:
Reordered because the lines were changed either ways — `sourcemaps` is longer than any other name there.